doc/rst/developer/chips/inactive/18.rst

```diff
@@ -17,6 +17,14 @@ Abstract
 Place array elements in NUMA-localized memory so that Chapel programs
 can take advantage of NUMA performance in the architecture.
 
+Update
+------
+
+This document describes an approach to NUMA support that was tried but
```

Comment:
This isn't accurate.  The document describes all the approaches for NUMA we thought of.  One is indeed the multi-ddata technique that was tried but led to performance problems, but another described in [this section](https://github.com/chapel-lang/chapel/blob/master/doc/rst/developer/chips/18.rst#single-ddata-with-separate-arrays) is what we adopted on XC with comm=ugni and mem=jemalloc, to good if not perfect effect.  So it's probably best just to say that we've implemented some of the techniques described and adopted a subset of them, and don't have (as far as I know) any plans to do much more.